assets/quick-add.css

```diff
@@ -35,6 +35,18 @@
   transform: none;
 }
 
+.quick-add-modal .quick-order-list__container {
+  padding-bottom: 1.5rem;
+}
+
+.quick-add-modal .link.product__view-details {
```

Comment:
This will change the regular quick add modal too. I think we only want this only for the bulk modal?